keycloak/import/hmda-realm.json

```diff
@@ -205,7 +205,7 @@
   "groups" : [ ],
   "defaultRoles" : [ "uma_authorization", "offline_access" ],
   "requiredCredentials" : [ "password" ],
-  "passwordPolicy" : "hashIterations(20000)",
+  "passwordPolicy" : "hashIterations and forceExpiredPasswordChange(30) and passwordHistory(12) and upperCase and lowerCase and digits and specialChars and length(12) and notUsername",
```

Comment:
Can we set `forceExpiredPasswordChange(30)` to 90 days instead? It would then basically cover a full filing period.